gtda/externals/python/cubical_complex_interface.py

```diff
@@ -95,13 +95,13 @@ def persistence(self, homology_coeff_field=11, min_persistence=0):
         if self.pcohptr is not None:
             del self.pcohptr
         if self.thisptr is not None:
-            pass
             self.pcohptr = Cubical_complex_persistence_interface(self.thisptr,
                                                                  True)
         persistence_result = []
         if self.pcohptr is not None:
-            persistence_result = self.pcohptr.get_persistence(
-                homology_coeff_field, min_persistence)
+            self.pcohptr.compute_persistence(homology_coeff_field,
```

Comment:
@MonkeyBreaker related to my question above.

Reply:
Same as before